### ISSUE TITLE
Remember login even after restarting Weblate

### ISSUE THREAD

**Issue by vroad** [WeblateOrg/weblate#3012]
**Is your feature request related to a problem? Please describe.**
I'm forced to re-login after restarting weblate, which is running as a docker container on Amazon ECS.
The server running weblate automatically starts and stops at certain time to save its cost, because it doesn't need to run 24h/365 days. I'm utilizing AWS's scheduled actions for AutoScaling groups for this.

Slight config change by env var causes ECS to restart container, and will also force users to re-login, which isn't nice either.

**Describe the solution you'd like**
Weblate should persist login session to somewhere, so that it can remember login even after restarts.
It should only ask login detail again only when a user doesn't use Weblate for long time.

**Describe alternatives you've considered**
Just re-login every day, pasting password with password manager extensions for Chrome.

**Additional context**

**Comment by nijel** [comment]
The login should stay. Most likely it's configuration issue in your setup. Maybe redis is not configured to persist data, what leads to losing sessions?

**Comment by vroad** [comment]
@nijel I've tried adding `--appendonly yes` to my redis server configuration. Still doesn't work. If it's the problem with redis config, then Weblate should not lose session in case only Weblate is restarted. Weblate always loses login sessions even if I restart only Weblate.

**Comment by vroad** [comment]
Well, I've probably found the cause. Only login session for the admin user is lost, and If I don't set `WEBLATE_ADMIN_PASSWORD`, login session for admin is also kept. Other users could keep login sessions.

Doc says that password is auto-generated on first startup, so I don't need to store this by myself.
I just need to login with auto-generated password and change it by myself.

https://docs.weblate.org/en/weblate-2.10/admin/deployments.html#envvar-WEBLATE_ADMIN_PASSWORD

So, this is not an issue of Weblate?
But I thought that this env var sets admin password only once, not on every startup, like official MySQL image's `MYSQL_ROOT_PASSWORD`.
Does it make sense to allow changing password on every startup?

**Comment by nijel** [comment]
Thank you for your report, the issue you have reported has just been fixed. The password is indeed changed and that leads to session invalidation.

* In case you see problem with the fix, please comment on this issue. 
* In case you see similar problem, please open separate issue.
* If you are happy with the outcome, consider supporting Weblate by [donating](https://weblate.org/donate/).
